README.md

```diff
@@ -1,9 +1,8 @@
-# Context Template Repository _(context-template)_
+# Ed25519Signature2020 Suite JSON-LD Context _(ed25519-signature-2020-context)_
 
-<!-- [![NPM Version](https://img.shields.io/npm/v/context-template.svg?style=flat-square)](https://npm.im/context-template) -->
-
-> A GitHub template repository for JSON-LD contexts.
+<!-- [![NPM Version](https://img.shields.io/npm/v/ed25519-signature-2020-context.svg?style=flat-square)](https://npm.im/ed25519-signature-2020-context) -->
```

Comment:
This will be broken until it's published on NPM, but might as well enable it now.  Also I think it's easier to just use the default style unless we're trying to match some other badges.
```suggestion
[![NPM Version](https://img.shields.io/npm/v/ed25519-signature-2020-context.svg)](https://npm.im/ed25519-signature-2020-context)
```